drivers/usb/device/usb_dc_stm32.c

```diff
@@ -55,10 +55,42 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(usb_dc_stm32);
 
-#if defined(DT_USB_BASE_ADDRESS) && defined(DT_USB_HS_BASE_ADDRESS)
+#if defined(DT_COMPAT_ST_STM32_OTGFS) && defined(DT_COMPAT_ST_STM32_OTGHS)
```

Comment:
DT_COMPAT_ST_STM32_USB seems also exclusive

Reply:
Indeed, but this error was made specifically for one series in which both versions are available in device tree description and where we should take care they are not activated at the same time.
This is not possible for other compatible ("st, stm32-usb") as its presence is already exclusive in dts files.